cleanlab_studio/internal/tlm/validation.py

```diff
@@ -255,3 +255,33 @@ def process_response_and_kwargs(
         {key: value for key, value in zip(combined_response_keys, values)}
         for values in combined_response_values_transposed
     ]
+
+
+def validate_tlm_hybrid_score_options(score_options: Any) -> None:
+    validate_tlm_options(score_options)
+
+    INVALID_SCORE_OPTIONS = {"num_candidate_responses"}
+
+    invalid_score_keys = set(score_options.keys()).intersection(INVALID_SCORE_OPTIONS)
+    if invalid_score_keys:
+        raise ValidationError(
+            f"Invalid keys in options dictionary for TLMHybrid: {invalid_score_keys}.\n"
+        )
+
+
+def get_tlm_hybrid_prompt_options(score_options: Any, response_model: str) -> Dict[str, Any]:
+    VALID_PROMPT_OPTIONS = {"max_tokens"}
+
+    if response_model not in _VALID_TLM_MODELS:
+        raise ValidationError(
+            f"{response_model} is not a supported model, valid models include: {_VALID_TLM_MODELS}"
+        )
+
+    prompt_options = {"model": response_model, "log": ["perplexity"]}
```

Comment:
just curious: why is this called `prompt_options` instead of `response_options`

Reply:
In my head it was "the options used in the `prompt` method", hence `prompt_options`, but I agree that `response_options` matches the naming convention better (given `response_model`), can update